setup.py

```diff
@@ -18,7 +18,10 @@
     url="https://github.com/ome/omero-certificates",
     packages=["omero_certificates", "omero.plugins"],
     setup_requires=["setuptools_scm"],
-    install_requires=["omero-py>=5.6.0"],
+    install_requires=[
+        "omero-py>=5.6.0",
+        "cryptography>=36.0.0",
```

Comment:
Tried to upgrade `omero-certificates` on a server with `cryptography==37.0.4` resulted in 

```
  File "/opt/omero/OMERO.venv/lib/python3.8/site-packages/omero_certificates/certificates.py", line 135, in create_certificates
    PrivateFormat.PKCS12.encryption_builder()
  File "/usr/lib/python3.8/enum.py", line 384, in __getattr__
    raise AttributeError(name) from None
AttributeError: PKCS12
```

As per https://cryptography.io/en/latest/hazmat/primitives/asymmetric/serialization/#cryptography.hazmat.primitives.serialization.PrivateFormat.PKCS12, this should become 

```suggestion
        "cryptography>=38.0.0",
```